cmd/vgmanager/vgmanager.go

```diff
@@ -197,9 +196,6 @@ func run(cmd *cobra.Command, _ []string, opts *Options) error {
 			return fmt.Errorf("could not add topolvm metrics: %w", err)
 		}
 
-		if err := os.MkdirAll(topolvm.DeviceDirectory, 0755); err != nil {
```

Comment:
Why don't we need this anymore?

Reply:
Because of https://github.com/topolvm/topolvm/releases/tag/v0.33.0 getting rid of the topolvm device directory

Reply:
Concretely, this is the change: https://github.com/topolvm/topolvm/pull/930/files